src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -125,6 +125,16 @@ class BaseOptionsSelector extends Component {
     }
 
     componentDidUpdate(prevProps) {
+        if (this.props.autoFocus && !prevProps.isFocused && this.props.isFocused) {
+            InteractionManager.runAfterInteractions(() => {
+                // Focus text input
```

Comment:
Comment should say _why_, on top of _what_, here's a suggestion, but feel free to add your own update
```suggestion
                // If we automatically focus on a text input when mounting a component, 
                // let's automatically focus on it when the component updates as well (eg, when navigating back from a page)
```